patches/minecraft/net/minecraft/CrashReport.java.patch

```diff
@@ -51,3 +52,12 @@
        stringbuilder.append("// ");
        stringbuilder.append(getErrorComment());
        stringbuilder.append("\n\n");
+@@ -123,7 +_,7 @@
+       for(int i = 0; i < 87; ++i) {
+          stringbuilder.append("-");
+       }
+-
++      net.minecraftforge.logging.CrashReportAnalyser.analyseCrashReport(this.exception, this.uncategorizedStackTrace);
```

Comment:
Why is this hooked in on this line? Honestly, looing at it, It looks like you should just make the CrashReportAnalyser class have a single entry point so that there is only a one line hook needed in this class. And we don't have to deal with the caching hooks elseware.